### PR TITLE
Save Notes Without Changing Task State

### PR DESCRIPTION
**Pull request by goldjay**
[FEV-1479](https://auderenow.atlassian.net/browse/FEV-1479)

* Button on Auditor panel to save current notes

### CODE REVIEW

File: src/Screens/TaskPanel.tsx [hunk 1]
```diff
@@ -591,17 +591,13 @@ class DetailsWrapper extends React.Component<
...
-      if (!result.success) {
-        this.setState(state => ({
-          buttonsBusy: {
-            ...state.buttonsBusy,
-            [key]: false
-          }
-        }));
-        return;
-      }
-      task = result.task || task;
```
Comment (balderdash) [inline src/Screens/TaskPanel.tsx]: `_onApprove()` in the auditor panel returns a modified task object with the visible entries marked as "reviewed", which is why I set the `task` variable here. Can you revert this change?

File: src/Screens/AuditorPanel.tsx [hunk 2]
```diff
@@ -318,7 +319,6 @@ export class AuditorDetails extends React.Component<
         phone: entry.phone,
         item: entry.item,
         "claimed cost": entry.claimedCost,
-        notes: (entry as any).notes || "",
```
[the remark this answers is not in the document]
Comment (goldjay) [reply src/Screens/AuditorPanel.tsx]: Global replace gone wrong 😅
